app/src/main/java/org/fossasia/openevent/app/core/settings/LegalPreferenceFragment.java

```diff
@@ -0,0 +1,70 @@
+package org.fossasia.openevent.app.core.settings;
+
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.preference.PreferenceManager;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.takisoft.fix.support.v7.preference.PreferenceFragmentCompat;
+
+import org.fossasia.openevent.app.R;
+import org.fossasia.openevent.app.common.Constants;
+import org.fossasia.openevent.app.ui.ViewUtils;
+
+public class LegalPreferenceFragment extends PreferenceFragmentCompat {
+
+    public static final String EVENTYAY_BASE_URL = "https://eventyay.com/";
+    public static final String PRIVACY_POLICY_URL = "privacy-policy/";
+    public final String cookiePolicyUrl = "cookie-policy/";
+    public static final String TERMS_OF_USE_URL = "terms/";
```

Comment:
Write full URL in each field, the duplication I was talking about was of the fields, remove base url and make all URLs fully qualified. Make cookiePolicyUrl static as well